server/src/main/java/io/crate/analyze/DropTableAnalyzer.java

```diff
@@ -63,16 +70,35 @@
                                                                boolean dropIfExists,
                                                                SessionContext sessionContext) {
         T tableInfo;
+        RelationName tableName;
+        boolean maybeCorrupt = false;
         try {
             //noinspection unchecked
             tableInfo = (T) schemas.resolveTableInfo(name, Operation.DROP, sessionContext.sessionUser(), sessionContext.searchPath());
+            tableName = tableInfo.ident();
         } catch (SchemaUnknownException | RelationUnknown e) {
             if (dropIfExists) {
                 tableInfo = null;
+                tableName = RelationName.of(name, sessionContext.searchPath().currentSchema());
             } else {
                 throw e;
             }
+        } catch (OperationOnInaccessibleRelationException e) {
+            throw e;
+        } catch (Throwable t) {
+            if (!sessionContext.sessionUser().isSuperUser()) {
+                throw t;
+            }
+            tableInfo = null;
+            maybeCorrupt = true;
+            tableName = RelationName.of(name, sessionContext.searchPath().currentSchema());
+            LOGGER.info(
```

Comment:
I'd turn this into debug as I think this is not relevant for the user. The table is dropped afterwards anyway so its not possible to do anything after seeing this log. Any other reason why informing the user is helpful here?

Reply:
> Any other reason why informing the user is helpful here

Because the cluster was in a weird state that caused the error. May be helpful for us to have that in the logs.